### ISSUE TITLE
bountysource.com/fundraisers is dead and the about link is sad

### ISSUE THREAD

**Issue by Huge** [creationix/js-git#146]
https://www.bountysource.com/fundraisers/325-js-git as well as any of the /fundraisers functionality is abandoned and no more hosted there. Would you link to an archived page, or some description file?

**Comment by Huge** [comment]
Weird as I just found https://www.kickstarter.com/projects/creationix/js-git nicely available as linked in https://github.com/creationix/js-git/blob/master/BACKERS.md . 
I suggest to put that one as the URL of this repo.

**Comment by TheKnarf** [comment]
Since neither is currently in use I suggest we just remove the links.

I don't seem to have access to change the project description / links, can you do it @creationix ?

![Screenshot 2020-11-23 at 11 34 50](https://user-images.githubusercontent.com/213777/99952408-e9b54680-2d7f-11eb-8153-c6e333332491.png)

**Comment by creationix** [comment]
Sorry I've not been checking my notifications for a while (life has been crazy).  Let me see if I can change it.

**Comment by creationix** [comment]
BTW, not sure if we want to link to it, but I made a similar project in Lua a while back (used by luvit.io's package manager).  Just yesterday I used the lua version to tweak one of my local repos.

https://gist.github.com/creationix/31828cf1056a9e70db8399be7c33a4ee

**Comment by Huge** [comment]
Cool, but I was just curious if the git stack is deployable in client JS inside a web browser.

**Comment by creationix** [comment]
JS-Git can run in browsers and that was the primary use case.  The problem that killed the motivation for me was Github refused to add CORS headers to the git endpoints and instead insisted I use their slow and buggy proprietary API.

So JS-Git can only talk to github if your code is hosted on github.com itself!  There are some workarounds, but they have less impact and use.

- Make your users install an extension that adds the CORS headers (many consider this a security risk)
- Host your own git repos and add the CORS headers yourself (at this point, you could use even better protocols than git smart http such as git over websocket)
- Convince the smaller hosting companies to add CORS headers. At one point gitlab offered to do this, but I don't think they ever did and I've seen very little demand for that anyway.
- Use a non-browser environment such as node or chrome packaged apps. (These environments have better ways already such as libgit2 or are deprecated like chrome apps)

TL/DR, it's not worth it for the majority of use cases.  I gave up years ago and got a paying job since I have a family to feed and all my work in this was basically worthless because github refused to cooperate.
